sdkv2.0.0/sdkv2.0.0/MojioSDK/MojioAuth.swift

```diff
@@ -8,26 +8,30 @@
 
 import UIKit
 import KeychainSwift
+import Alamofire
+import SwiftyJSON
 
 class MojioAuth: NSObject, AuthControllerDelegate {
     
-    var appId : String
-    var redirectURL : String
+    var appId : String?
+    var redirectURL : String?
     var loginURL : NSURL!
     var loginCompletion : (Void) -> (Void)
     var authController : AuthViewController?
-    
-//    static let authClient = MojioAuth(appId: <#T##String#>, redirectURI: <#T##String#>)
-    
+        
     init(appId : String, redirectURI : String) {
-        self.appId = appId;
-        self.redirectURL = redirectURI;
+        self.appId = appId
+        self.redirectURL = redirectURI
         
-        let loginString = String (format: "https://staging-accounts.moj.io/oauth2/authorize?response_type=token&redirect_uri=%@&client_id=%@&scope=full", self.redirectURL, self.appId);
+        let accountsEndpoint = MojioClientEnvironment.clientEnvironment.getAccountsEndpoint()
```

Comment:
It looks like I will end up rewriting this class almost entirely. JZ is not a huge fan of this approach either.